src/js/common/schemaform/widgets/RadioWidget.jsx

```diff
@@ -1,19 +1,29 @@
 import React from 'react';
 
+import ExpandingGroup from '../../components/form-elements/ExpandingGroup';
+
 export default function RadioWidget({
   options,
   value,
   disabled,
   onChange,
   id
 }) {
-  const { enumOptions, labels = {} } = options;
+  const { enumOptions, labels = {}, nestedContent = {} } = options;
+
+  // nested content could be a component or just jsx/text
+  let content = nestedContent[value];
+  if (typeof content === 'function') {
```

Comment:
Is this the best way to test if a variable is a React component? I haven't found a better one... :confused: